java/change-notes/2021-08-05-jodd-unsafe-deserialization.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
```

Comment:
Committed by accident? It's also in [its own PR](https://github.com/github/codeql/pull/6434/files#diff-7a3164e71b621cbdafe775f66e4049581ff08974a4d2df92cfaf3eb9c4088a3b).

Reply:
This PR is based on that one; in this PR please only review the top commits relating to Flexjson